doc/changelog/03-notations/11948-hexadecimal.rst

```diff
@@ -0,0 +1,12 @@
+- **Added:**
+  Numeral notations now parse hexadecimal constants such as ``0x2a``
+  or ``0xb.2ap-2``.  Parsers added for nat, positive, Z, N, Q, R,
+  primitive integers and primitive floats.
```

Comment:
I believe the standard convention is to not have `.` between the sentence and the parenthetical
```suggestion
  primitive integers and primitive floats
```

Reply:
Makes sense